pysqldb3/pysqldb3.py

```diff
@@ -1779,19 +1779,19 @@ def df_to_geojson(df):
         fig.show()
         return
 
-    def query_to_shp(self, query, path=None, shp_name=None, cmd=None, gdal_data_loc=GDAL_DATA_LOC,
-                     print_cmd=False, srid=2263, shp = True, gpkg_tbl = None):
+    def query_to_shp(self, query,  gpkg_tbl, path=None, shp_name=None, cmd=None, gdal_data_loc=GDAL_DATA_LOC,
```

Comment:
1st why is there here, i see a seperate function query_to_gpkg? 2nd if it is going to be in this then the gpkg_tbl needs to be an optional param